core/src/main/java/com/amazon/opendistroforelasticsearch/sql/expression/operator/predicate/BinaryPredicateOperator.java

```diff
@@ -163,6 +168,49 @@ public static void register(BuiltinFunctionRepository repository) {
           .put(LITERAL_MISSING, LITERAL_MISSING, LITERAL_TRUE)
           .build();
 
+  /**
+   * The notEqualTo logic.
+   * A       B       A != B
+   * NULL    NULL    FALSE
+   * NULL    MISSING TRUE
+   * MISSING NULL    TRUE
+   * MISSING MISSING FALSE
+   */
+  private static Table<ExprValue, ExprValue, ExprValue> notEqualTable =
+      new ImmutableTable.Builder<ExprValue, ExprValue, ExprValue>()
+          .put(LITERAL_NULL, LITERAL_NULL, LITERAL_FALSE)
+          .put(LITERAL_NULL, LITERAL_MISSING, LITERAL_TRUE)
+          .put(LITERAL_MISSING, LITERAL_NULL, LITERAL_TRUE)
+          .put(LITERAL_MISSING, LITERAL_MISSING, LITERAL_FALSE)
+          .build();
+
+  /**
```

Comment:
The comments is not very related to the valueComparisonTable defined in here. If there is no special handling for NULL and MISSING, the behavior should follow the generic rule defined in here. https://asterixdb.apache.org/docs/0.9.3/sqlpp/manual.html#Operator_expressions

Reply:
Looks good, I will refer to this manual and make corresponding changes, thanks!